nexus/types/src/external_api/params.rs

```diff
@@ -1600,6 +1620,53 @@ pub struct Route {
     pub local_pref: Option<u32>,
 }
 
+/// A route to a destination network through a gateway address to add or
+/// remove to an interface.
```

Comment:
suggestion: "to add to or remove from", although this still needs a bit of work to make more clear

Reply:
Most of the language in that docstring is unnecessary since the struct fields provide those details, so I'm going to simplify it